texts/doi/export.yml

```
@@ -0,0 +1,3 @@
+title: Export
+description: Download all your {{count}} DOI records as a CSV.
```

Perhaps the text should be different. 
```suggestion
description: Download {{count}} new DOI records as a CSV.
```